sqlglot/optimizer/optimizer.py

```diff
@@ -9,6 +9,9 @@
 from sqlglot.optimizer.qualify_columns import qualify_columns
 from sqlglot.optimizer.qualify_tables import qualify_tables
 from sqlglot.optimizer.quote_identities import quote_identities
+from sqlglot.optimizer.rewrite_derived_tables_as_ctes import (  # noqa
```

Comment:
what's thi lint for

Reply:
Since it's an unused import.

I'll just remove this import.